source/user/lo.rst

```diff
@@ -0,0 +1,66 @@
+.. _user_lo:
+
+**********************
+Localized Orbitals
+**********************
+
+*Modules*: :mod:`lo`
+
+Introduction
+============
+A molecular orbital is usually delocalized, i.e. its spatial distribution is
+all over the whole molecular rather than localized around several atoms or bonds.
+However, one can choose a proper unitary rotation :math:`U`
+
+.. math::
+
+    \phi = \psi U
+
+such that the resulting
+orbitals :math:`\phi` are as local as possible. This is typically achieved by two kinds of
+methods: One is by projection onto a pre-defined local orbital set
+(e.g., AO, pseudopotential orbitals); The other is by numerical optimization on
+a cost function :math:`f`, which can be chosen as different localization
+criterion, e.g., in Boys localization, the dipole (spread) of a molecule,
+
+.. math::
+
+    f(U) = \langle\psi|r^2|\psi\rangle - \langle\psi|r|\psi\rangle^2
+
+
```

Comment:
The section should also discuss Edmiston-Ruedenberg, as well as the Pipek-Mezey methods (original method based on Mulliken charges; Löwdin charges; IAOs; as well as the generalized Pipek-Mezey method which is also implemented in the `PipekMezey` class).

Reply:
Ok - just two more equations, I guess ...?

Reply:
Yes!

Reply:
You can refer to my recent paper that is cited in the source for discussion on the Pipek-Mezey methods.

Reply:
Perfect.